src/routes/auth.js

```diff
@@ -146,9 +146,11 @@ export default function authRoute() {
   }));
 
   router.put('/national', authenticateJWT, checkAuthenticated, asyncHandler(async (req, res) => {
-    const { first_name, last_name } = await prisma.users.findUnique({ where: { uuid: req.user.uuid } });
+    const { first_name, last_name, club_uuid } = await prisma.users.findUnique({ where: { uuid: req.user.uuid } });
     await update(req.user.uuid, req.body);
 
+    const { name } = await getClub(club_uuid);
```

Comment:
I would rename this as I think it's too easy to get confused down the line especially at the email layer, where this just comes through without context

```suggestion
    const { name: club_name } = await getClub(club_uuid);
```